certbot/renewal.py

```diff
@@ -304,6 +304,8 @@ def _renew_describe_results(config, renew_successes, renew_failures,
         notify(report(renew_skipped, "skipped"))
     if not renew_successes and not renew_failures:
         notify("No renewals were attempted.")
+        if config.pre_hook is not None or config.post_hook is not None:
```

Comment:
Renew hooks are specifically run after each cert is renewed. With this understanding, I personally wouldn't expect a renew hook to be run if no certs are renewed.

Pre and post hooks on the other hand are respectively run before or after attempting renewal. To me, it isn't as clear whether or not I should expect Certbot to run them if nothing needs to be renewed.

I'm fine making this change though. Just waiting on tests to finish locally.